src/d/actor/d_a_steam_tag.cpp

```diff
@@ -4,56 +4,209 @@
 //
 
 #include "d/actor/d_a_steam_tag.h"
+#include "JSystem/J3DGraphBase/J3DSys.h"
+#include "JSystem/JParticle/JPAEmitter.h"
+#include "d/d_cc_d.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_item.h"
+#include "d/d_item_data.h"
 #include "d/d_procname.h"
+#include "dolphin/mtx/vec.h"
+
+static dCcD_SrcCps l_cps_src = {
+    // dCcD_SrcGObjInf
+    {
+        /* Flags             */ 0,
+        /* SrcObjAt  Type    */ AT_TYPE_WIND,
+        /* SrcObjAt  Atp     */ 0,
+        /* SrcObjAt  SPrm    */ cCcD_AtSPrm_Set_e | cCcD_AtSPrm_GrpAll_e,
+        /* SrcObjTg  Type    */ 0,
+        /* SrcObjTg  SPrm    */ 0,
+        /* SrcObjCo  SPrm    */ 0,
+        /* SrcGObjAt Se      */ 0,
+        /* SrcGObjAt HitMark */ 0,
+        /* SrcGObjAt Spl     */ 0,
+        /* SrcGObjAt Mtrl    */ 0,
+        /* SrcGObjAt SPrm    */ dCcG_AtSPrm_NoHitMark_e,
+        /* SrcGObjTg Se      */ 0,
+        /* SrcGObjTg HitMark */ 0,
+        /* SrcGObjTg Spl     */ 0,
+        /* SrcGObjTg Mtrl    */ 0,
+        /* SrcGObjTg SPrm    */ dCcG_TgSPrm_NoHitMark_e,
+        /* SrcGObjCo SPrm    */ 0,
+    },
+    // cM3dGCpsS
+    {
+        /* P0 */ 0.0f, 0.0f, 0.0f,
+        /* P1 */ 0.0f, 0.0f, 0.0f,
+        /* Height */ 200.0f,
+    },
+};
+
+daSteamTag_mData daSteamTag_c::mData = {15861, 49807, 300, 100, 50, 150, 0xB4, 0x00, 0x00};
 
 /* 00000078-00000084       .text getData__12daSteamTag_cFv */
-void daSteamTag_c::getData() {
-    /* Nonmatching */
+const daSteamTag_mData* daSteamTag_c::getData() {
+    return &daSteamTag_c::mData;
 }
 
 /* 00000084-0000029C       .text CreateInit__12daSteamTag_cFv */
-void daSteamTag_c::CreateInit() {
-    /* Nonmatching */
+s32 daSteamTag_c::CreateInit() {
+    m29B = daSteamTag_prm::getSchBit(this);
+    float fVar2;
+    mEmitTimer = getData()->mEmitTimer_offset + cM_rndF(getData()->mEmitTimer_seed);
+    mCreateTimer = cM_rndF(getData()->mCreateTimer_seed);;
+    mpEmitter = NULL;
+    mEmitTimer = 0;
+    mGStts.Init(0xFF,0xFF,this);
+    mCps.Set(l_cps_src);
+    mCps.SetStts(&mGStts);
+    mPntWindCpsS.mStart.x = current.pos.x;
+    mPntWindCpsS.mStart.y = current.pos.y;
+    mPntWindCpsS.mStart.z = current.pos.z;
+    mDoMtx_stack_c::transS(current.pos);
+    mDoMtx_stack_c::ZXYrotM(current.angle);
+    fVar2 = 200.0f;
+    if (current.angle.x != 0) {
+      fVar2 = 50.0f;
+    }
+    mDoMtx_stack_c::transM(0.0f, fVar2, 0.0f);
+    mDoMtx_stack_c::multVec(&cXyz::Zero, &mPntWindCpsS.mEnd);
+    mPntWindCpsS.mRadius = 50.0;
+    mCps.cM3dGCps::Set(mPntWindCpsS);
+    mCps.CalcAtVec();
+    mPointWind.set_pwind_init(&mPntWindCpsS);
+    fopAcM_offDraw(this);
+    return 1;
```

Comment:
```suggestion
    return TRUE;
```

CreateInit usually returns `BOOL` (don't confuse with Create, which returns a phase state)